install/kubernetes/templates/helm/istio/values.yaml.tmpl

```diff
@@ -1,31 +1,45 @@
-istio:
-   deploy_base_config: true
-   initializer_enabled: true
-   mixer_enabled: true
-   pilot_enabled: true
-   ca_hub: {CA_HUB}
-   ca_tag: {CA_TAG}
-   ca_demo: false
-   ingress:
+global:
+  namespace: istio-system
```

Comment:
Since this is a template file I'll mention that in other template files the namespace is set by updateVersion.sh, where the name ultimately comes from this line in istio.VERSION:
export ISTIO_NAMESPACE="istio-system"

Reply:
Thanks, agreed!  at the moment, we can't really let users to customize the Istio namespace.  If that is something of interesting, perhaps it needs to be tested and submit in a different PR.

Reply:
I suppose adding {ISTIO_NAMESPACE} wouldn't hurt ?